cli/azd/pkg/tools/azcli/appconfig.go

```diff
@@ -0,0 +1,83 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appconfiguration/armappconfiguration"
+	"github.com/azure/azure-dev/cli/azd/pkg/convert"
+	"github.com/azure/azure-dev/cli/azd/pkg/identity"
+)
+
+type AzCliAppConfig struct {
+	Id         string `json:"id"`
+	Name       string `json:"name"`
+	Location   string `json:"location"`
+	Properties struct {
+		EnablePurgeProtection bool `json:"enablePurgeProtection"`
+	} `json:"properties"`
+}
+
+type AzCliAppConfigSecret struct {
+	Id    string `json:"id"`
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+func (cli *azCli) GetAppConfig(ctx context.Context, subscriptionId string, resourceGroupName string, configName string) (*AzCliAppConfig, error) {
+	client, err := cli.createAppConfigClient(ctx, subscriptionId)
```

Comment:
might be good to call the client `appConfigStoresClient`, as that's the client we are creating.
It gives a hint about the resource and operation that it will fetch from the service